util.rb

```diff
@@ -4,58 +4,48 @@
 # IAM - a resource usage metric collection and reporting system
 class Util
   def self.max_value(data)
-    if data.empty?
-      return 0
-    end
+    return 0 if data.empty?
     max = data[0][:value]
     data.each do |item|
       compare_max = item[:value]
-      if max < compare_max
-        max = compare_max
-      end
+      max = compare_max if max < compare_max
     end
-    return max
+    max
   end
 
   def self.min_value(data)
     return 0 if data.empty?
     min = data[0][:value]
     data.each do |item|
       compare_min = item[:value]
-      if min > compare_min
-        min = compare_min
-      end
+      min = compare_min if min > compare_min
     end
-    return min
+    min
   end
 
   def self.average_value(data)
-    if data.empty?
-      return 0
-    end
+    return 0 if data.empty?
     average = data[0][:value]
     count = 1
     data.slice(1..-1).each do |item|
       average += item[:value]
       count += 1
     end
     average /= count
-    return average
+    average
   end
 
-  def self.is_positive_integer_value(data)
+  def self.positive_integer_value?(data)
     flag = true
     i = 0
     while i < data.length && flag != false
       if data[i][:value].is_a? Integer
-        if data[i][:value] < 0
-          flag = false
-        end
+        flag = false if data[i][:value] < 0
       else
         flag = false
       end
       i += 1
     end
-    return flag
+    flag
```

Comment:
I was taught to only use multiple return statements in code if I'm checking something right at the beginning, such as line 7 when checking to see if the list is empty. Otherwise, I was taught to not use multiple return statements, reasoning being that it is easier to debug.